packages/sogrim-app/src/components/Banner/BannerCards/DegreeMainStatus.tsx

```diff
@@ -1,60 +1,146 @@
-import { Button, Card, CardContent, Typography } from "@mui/material";
+import {
```

Comment:
we need to think about break out all the logic - and maybe create new sub-store for the `DegreeMainStatus`

Reply:
I will open a new issue for that.
It's a change which shouldn't be done in the scope of this task.

Reply:
I will open a new issue for that.
It's a change which shouldn't be done in the scope of this task.